modules/rosa-classic-cluster-terraform-file-creation.adoc

```diff
@@ -0,0 +1,297 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-classic-creating-a-cluster-quickly-terraform.adoc
+//
+
+ifeval::["{context}" == "rosa-classic-creating-a-cluster-quickly-terraform"]
+:tf-defaults:
+endif::[]
+:_content-type: PROCEDURE
+
+[id="rosa-classic-cluster-terraform-file-creation_{context}"]
+= Creating your Terraform files locally
+
+After you set up your link:https://console.redhat.com/openshift/token[offline {cluster-manager-first} token], you need to create the Terraform files locally to build your cluster. You can create these files by using the following code templates.
```

Comment:
Should we recommend https://console.redhat.com/openshift/token/rosa instead of https://console.redhat.com/openshift/token? It is the same token, but the page is more specific to rosa rather than ocm.